components/main/main.component.tsx

```diff
@@ -65,6 +60,11 @@ export const MainComponent: React.FC<MainComponentProps> = ({
     setOpenDropdown(null)
   }
 
+  const handleContinue = () => {
+    const amount = control._formValues.amount
+    onContinueClick({ amount, fromNetwork, toNetwork })
```

Comment:
`amount`, `fromNetwork`, `toNetwork` is available in the parent component so there is no point to send it back as arg.